src/content/guides/prepare-azure-subscription-for-guest-clusters/index.md

```diff
@@ -0,0 +1,119 @@
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+#### 2. Role definition
+
+Download our [Role definition template](https://raw.githubusercontent.com/giantswarm/azure-operator/master/policies/guest.tmpl.json).
+
+To find out your subscription ID you can use [the Azure portal](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade), as shown in the screenshot below:
+
+![Azure Subscriptions list](/img/azure-subscriptions-list.png)
+
+Alternatively you can use the Azure CLI like so:
+
+```nohighlight
+$ az account list --output table
+Name     CloudName    SubscriptionId                        State    IsDefault
+-------  -----------  ------------------------------------  -------  -----------
+Example  AzureCloud   6ec148b8-8bea-4dd3-82bc-1787c8260e4a  Enabled  True
+```
+
+With the edited role definition in the file `guest.json`, create the role definition using the Azure CLI:
+
+```nohighlight
+$ az role definition create --role-definition @guest.json
+```
+
+On success this command prints the created role definition and can be ignored.
+
+#### 3. Service Principal
+
+Create the service principal using the Azure CLI:
+
+```nohighlight
+$ az ad sp create-for-rbac --name "azure-operator-sp" --role "azure-operator" --query '{client_id:appId, secret_key:password, tenant_id:tenant, subscription_id:""}'
+{
+    "client_id": "72bc3de4-3cf8-46c5-bd2b-243368ed0622",
+    "secret_key": "d6b2cb93-cae9-44b3-8ec5-dc5feb8c28ba",
+    "subscription_id": null,
+    "tenant_id": "31f75bf9-3d8c-4691-95c0-83dd71613db8"
+}
+```
+
+## Configure the Giant Swarm organization
+
+In the previous section, we explained how to create the Service Principal in the Azure subscription in order to run Giant Swarm guest clusters.
+
+Giant Swarm guest clusters are owned by organizations, which allows to control access to clusters, since only members of the owner organization have access to the management functions of a cluster.
+
+In order to run a guest cluster in your Azure subscription, the organization owning your cluster has to know about the Service Principal you just created.
+
+In the output from step 3 of the previous section you need to replace the `null` value in front of `subscription_id:` with your actual subscription ID. This document is the credential which needs to be registered within Giant Swarm API.
+
+```json
+{
+    "client_id": "72bc3de4-3cf8-46c5-bd2b-243368ed0622",
+    "secret_key": "d6b2cb93-cae9-44b3-8ec5-dc5feb8c28ba",
+    "subscription_id": "6ec148b8-8bea-4dd3-82bc-1787c8260e4a",
+    "tenant_id": "31f75bf9-3d8c-4691-95c0-83dd71613db8"
+}
+```
+
+If you have direct access to the Giant Swarm API, please follow the [documentation](https://docs.giantswarm.io/api/#operation/addCredentials) to set the credentials of your organization via the API.
```

Comment:
Here we tell to check out the documentation but we then provide an example on how to do it. I think that can be re-phrased better.